openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -2499,6 +2499,19 @@ def test_update_email_opt_in_inactive_user(self):
         )
         self.assertEquals(preference.value, u"True")
 
+    def test_update_email_opt_in_anonymous_user(self):
+        """
+        Test that an anonymous user gets 403 response when
+        updating email optin preference.
+        """
+        self.client.logout()
+        # Register, which should trigger an activation email
```

Comment:
This comment is confusing, looks like we are testing activation email functionality here but in actual we are testing opt-in. I don't think we need it here, remove it!